tests/e2e/tests/pilot/controller_test.go

```diff
@@ -99,20 +99,23 @@ func TestUnknownConfig(t *testing.T) {
 }
 
 func TestControllerEvents(t *testing.T) {
+	t.Skipf("Skipping %s as its flakey in CircleCI", t.Name())
```

Comment:
Please add link to the issue (https://github.com/istio/istio/issues/5338) that I just filed.

Same for the below two lines too.